packages/react-components/react-combobox/library/src/utils/useComboboxPositioning.ts

```diff
@@ -10,15 +10,11 @@ export function useComboboxPositioning(props: ComboboxBaseProps): [
 ] {
   const { positioning } = props;
 
-  // Set a default set of fallback positions to try if the dropdown does not fit on screen
-  const fallbackPositions: PositioningShorthandValue[] = ['above', 'after', 'after-top', 'before', 'before-top'];
```

Comment:
I think changing the fallbackPositions would need to be a larger conversation, since it'd be a breaking change. The current `fallbackPositions` values matches what we have in v8, which also allows the combobox/dropdown popup to render to the left or right of the trigger.

If we do change this, I think it should include `above` as a fallback to allow the popup to position above/below based on screen size. In general, I'm a bit leery of this change though, since it could make selection on short screens (and especially with zoom) a lot harder.

Teams can also currently override this through the `positioning` prop already.

Reply:
> Teams can also currently override this through the positioning prop already.

@smhigley FYI this is _partially_ true: it works in JS land, but we don't expose `fallbackPositions` in `positioning` prop - it will cause a TS error:

<img width="1050" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/62dd5159-b28c-4711-8a4e-6b8eb7212927">

Reply:
I exported `fallbackPositions` prop. Hope it's a right fix